src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -246,62 +382,115 @@ export class SkyRepeaterItemComponent implements OnDestroy, OnInit {
   }
 
   public onReorderHandleKeyDown(event: KeyboardEvent): void {
-    let key = event.key.toLowerCase();
-    if (key === ' ' || key === 'enter') {
-      this.keyboardReorderingEnabled = !this.keyboardReorderingEnabled;
-      this.reorderSteps = 0;
+    switch (event.key.toLowerCase()) {
+      case ' ':
+        this.keyboardToggleReorder();
+        event.preventDefault();
+        event.stopPropagation();
+        break;
+
+      case 'enter':
```

Comment:
You can move this `case` up to the "space" case, to avoid duplicate code.
```
case ' ':
case 'enter':
  // ...
  break;
```

Reply:
Didn't realize you could do that. Great tip. Done.